### ISSUE TITLE
Pro controller motion controls detected but not working in game

### ISSUE THREAD

**Issue by TheScentOfYourShampooOnTheFlowers** [yuzu-emu/yuzu#12866]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Mainline 1680

### Description of Issue

In the configuration the motion controls are working fine for my pro controller but in game (TOTK) they don't work.
D
![image_2024-01-30_175014998](https://github.com/yuzu-emu/yuzu/assets/143460577/820c7f2c-80b8-49eb-b1c1-fbab1e3dd6d3)

### Expected Behavior

Motion controls working in game when needed such as aiming a bow or using Ultrahand

### Reproduction Steps

Pro controller connected to Yuzu
Motion control set up
TOTK

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/14105198/yuzu_log.txt)

### System Configuration

CPU: 12th Gen Intel(R) Core(TM) i7-12700F
GPU/Driver: NVIDIA GeForce GTX 1660 SUPER
RAM: 16.0 GB
OS: Windows 11

**Comment by german77** [comment]
Does yuzu detect motion on the config window? The little rectangle moves with your controller?

**Comment by TheScentOfYourShampooOnTheFlowers** [comment]
Yep it does. I've tried with both Betterjoy and with just the controller. Both show that motion is detected but in game it doesn't work.

**Comment by german77** [comment]
Check game config. Maybe is disabled. As long as it works on the preview it should work in game. They display the same data
